pallets/capacity/src/lib.rs

```diff
@@ -1125,6 +1132,6 @@ impl<T: Config> StakingRewardsProvider<T> for Pallet<T> {
 
 	/// How much, as a percentage of staked token, to boost a targeted Provider when staking.
 	fn capacity_boost(amount: BalanceOf<T>) -> BalanceOf<T> {
-		Perbill::from_percent(5u32).mul(amount)
+		Perbill::from_percent(50u32).mul(amount)
```

Comment:
might be nice to know why 50%?

Reply:
@harry-evans , @wilwade do you have a specific answer? I could put it in a comment.

Reply:
I don't think we had a specific reason other than it felt like a good reasonable number. It has a fair shot at being updated before going to mainnet if we need to after some more modeling.